apps/company/serializers.py

```
@@ -248,34 +308,595 @@ class Meta:
-                    case 'yandex':
-                        results[vendor_code][warehouse] = stock.yandex_market_quantity
-                    case 'wildberries':
-                        results[vendor_code][warehouse] = stock.wildberries_quantity
-                    case _:
-                        results[vendor_code][warehouse] = stock.ozon_quantity + stock.wildberries_quantity + stock.yandex_market_quantity
+        shelfs = WarehouseHistory.objects.filter(product=obj.product)
+        for shelf in shelfs:
+            if shelf.shelf:
+                name = shelf.shelf.shelf_name
+            else:
+                name = "-"
```

Ruff

Use ternary operator `name = shelf.shelf.shelf_name if shelf.shelf else '-'` instead of `if`-`else`-block

Replace `if`-`else`-block with `name = shelf.shelf.shelf_name if shelf.shelf else '-'`

(SIM108)